modules/oxide/src/less/theme/components/toolbar-button/toolbar-split-button.less

```diff
@@ -40,23 +40,27 @@
     margin: 0;
   }
 
+  .tox-split-button.tox-tbtn--disabled:hover,
+  .tox-split-button.tox-tbtn--disabled:focus,
+  .tox-split-button.tox-tbtn--disabled .tox-tbtn:hover,
+  .tox-split-button.tox-tbtn--disabled .tox-tbtn:focus {
+    background: @toolbar-button-disabled-background-color;
+    box-shadow: @toolbar-button-disabled-box-shadow;
+    color: @toolbar-button-disabled-text-color;
+  }
```

Comment:
This doesn't seem related to this Jira so should it have been included? If so then it's not working as expected with keyboard navigation because the inner button never gets focus and its background overrides the parent one.

- [ ] Fix issues fully or split out into a separate Jira

Reply:
This maybe should also have a changelog entry if you keep it, since it looks to be trying to fix focus style issues when the button is disabled.

Reply:
Ignore this sorry, as @ltrouton pointed out on slack this was just moved from below instead.